components/src/dynamo/common/tests/multimodal/test_lazy_imports.py

```
@@ -0,0 +1,142 @@
+        assert mm.EMBEDDING_SENDER_FACTORIES is senders
+        assert mm.EMBEDDING_RECEIVER_FACTORIES is receivers
+        """
+    )
+
+
+def test_unknown_attribute_still_raises_attribute_error() -> None:
+    """``__getattr__`` must not swallow typos into silence."""
+    import dynamo.common.multimodal as mm
+
+    with pytest.raises(AttributeError, match="no attribute 'NotARealSymbol'"):
+        mm.NotARealSymbol
```

Ruff (0.16.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)